README.rst

```diff
@@ -45,6 +45,17 @@ elections server::
 Once that is running, simply go to http://localhost:5002/ in your browser on
 your host to see your running elections test instance.
 
+A note about Fonts
+~~~~~~~~~~~~~~~~~~
+Fedora elections uses webfonts pulled down from Fedora infrastructure that might
+not work when hacking locally due to CORS resctrictions. Installing the fonts on
+the system that you are looking at the elections website on will be the same as
```

Comment:
I specifically chose this language, as to make the distiction between the machine you are developing on (which could be the vagrant-based guest), and the host, which is the most likely that you are browsing to the website. For the fonts to show, you need to just have them installed on the machine that you are browsing to the test site on, not neccecarily the server that is hosting the elections site.